rerun_cpp/src/rerun/recording_stream.hpp

```diff
@@ -174,6 +174,16 @@ namespace rerun {
         /// This function returns immediately.
         Error save(std::string_view path) const;
 
+        /// Stream all log-data to standard output.
+        ///
+        /// Pipe the result into the Rerun Viewer to visualize it.
+        ///
+        /// If there isn't any listener at the other end of the pipe, the `RecordingStream` will
+        /// default back to `buffered` mode, in order not to break the user's terminal.
+        ///
+        /// This function returns immediately.
+        Error to_stdout() const;
```

Comment:
gosh this `to_stdout` business is annoying. But I have no better naming idea either :(

Reply:
I should add a comment though